FP_Factions/JSDF/config.cpp

```diff
@@ -147,16 +521,881 @@ class cfgvehicles
+		uniformClass = "FP_Uniform_BDU_02_Jietai";  
+		weapons[] = {"hlc_lmg_minimipara_long_railed","Throw","Put"};
+		respawnWeapons[] = {"hlc_lmg_minimipara_long_railed","Throw","Put"};
+		magazines[] = {"hlc_200rnd_556x45_M_SAW","hlc_200rnd_556x45_M_SAW","hlc_200rnd_556x45_M_SAW","HandGrenade","HandGrenade","SmokeShell","SmokeShellRed"};
+		respawnMagazines[] = {"hlc_200rnd_556x45_M_SAW","hlc_200rnd_556x45_M_SAW","hlc_200rnd_556x45_M_SAW","HandGrenade","HandGrenade","SmokeShell","SmokeShellRed"};
+		cost = 95000;
+		threat[] = {1,0.1,0.3};
+		icon = "iconManMG";
+	};
+	class FP_Faction_JSDF_Paratrooper_M: FP_Faction_JSDF_Paratrooper_BASE
+	{
+		class SpeechVariants
+		{
+			class Default
+			{
+				speechSingular[] = {"veh_infantry_sniper_s"};
+				speechPlural[] = {"veh_infantry_sniper_p"};
+			};
+		};
+		textSingular = "$STR_A3_nameSound_veh_infantry_sniper_s";
+		textPlural = "$STR_A3_nameSound_veh_infantry_sniper_p";
+		nameSound = "veh_infantry_sniper_s";
+		author = "FP Mod Team";
+		scope = 2;
+		scopeCurator = 2;
+		displayName = "Paratrooper Marksman";
+		weapons[] = {"hlc_rifle_m14dmr_optic_artel_m14","RH_p226","Throw","Put"};
+		respawnWeapons[] = {"hlc_rifle_m14dmr_optic_artel_m14","RH_p226","Throw","Put"};
+		magazines[] = {"hlc_20Rnd_762x51_mk316_M14","hlc_20Rnd_762x51_mk316_M14","hlc_20Rnd_762x51_mk316_M14","hlc_20Rnd_762x51_mk316_M14","hlc_20Rnd_762x51_mk316_M14","hlc_20Rnd_762x51_mk316_M14","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","HandGrenade","HandGrenade","SmokeShell","SmokeShellRed"};
+		respawnMagazines[] = {"hlc_20Rnd_762x51_mk316_M14","hlc_20Rnd_762x51_mk316_M14","hlc_20Rnd_762x51_mk316_M14","hlc_20Rnd_762x51_mk316_M14","hlc_20Rnd_762x51_mk316_M14","hlc_20Rnd_762x51_mk316_M14","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","HandGrenade","HandGrenade","SmokeShell","SmokeShellRed"};
+		cost = 150000;
+		Items[] = {"FirstAidKit","ACE_RangeCard"};
+		RespawnItems[] = {"FirstAidKit","ACE_RangeCard"};
+		sensitivity = 3;
+	};
+	class FP_Faction_JSDF_Paratrooper_MMG: FP_Faction_JSDF_Paratrooper_AR
+	{
+		displayName = "Paratrooper Machinegunner";  
+		weapons[] = {"rhs_weap_m240G","RH_p226","Throw","Put"};
+		respawnWeapons[] = {"rhs_weap_m240G","RH_p226","Throw","Put"};
+		magazines[] = {"rhsusf_50Rnd_762x51","rhsusf_50Rnd_762x51","rhsusf_50Rnd_762x51","rhsusf_50Rnd_762x51","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","HandGrenade","HandGrenade","SmokeShell","SmokeShellRed"};
+		respawnMagazines[] = {"rhsusf_50Rnd_762x51","rhsusf_50Rnd_762x51","rhsusf_50Rnd_762x51","rhsusf_50Rnd_762x51","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","HandGrenade","HandGrenade","SmokeShell","SmokeShellRed"};
+	};
+};
+
+class cfgWeapons
+{
+	class rhs_weap_m4a1_blockII_bk;
+	class rhs_weap_m4a1_blockII_M203_bk;
+	class rhs_weap_m4a1_blockII_bk_nt4_black_compm4_rvg_blk: rhs_weap_m4a1_blockII_bk
```

Comment:
when adding new guns, please use the `FP_` prefix so it is clear this addition is made by use and not rhs, hlc etc